### PR TITLE
Remove erroneous UNIMP fill of .bss section

### PR DESCRIPTION
**Pull request by luismarques**
The fill value of .balign is ignored for .bss. Binutils will warn about
using a non-zero fill value. Clang's integrated assembler won't complain
but it has a debug assertion that will trip in debug builds of the toolchain.

### CODE REVIEW

**Review by marnovandermaas — APPROVED** [review]
Looks good to me, it would be good to get someone else to approve it as well before merging.

**Review by pamaury — APPROVED** [review]
Looks for to me.

**Comment by marnovandermaas** [comment]
Ok, let's wait for CI to finish before merging.

**Review by HU90m — APPROVED** [review]

**Pull request merged**
